src/Umbraco.Web.UI.Client/src/views/webhooks/overlays/details.controller.js

```diff
@@ -21,6 +22,21 @@
       return obj;
     }
 
+    function detectLanguage(headers, defaultLanguage) {
+      const matches = headers.match(/^Content-Type:\s*(?<type>[a-z\/+.-]+)(\;?.*?)$/mi)
+      if (matches) {
+        const contentType = matches.groups["type"];
+        if (contentType === "application/json")
+          return "JSON";
+        if (contentType === "text/html")
+          return "HTML";
+        if (contentType === "application/xml" || contentType === "text/xml")
```

Comment:
Should we include `application/rss+xml` and `application/atom+xml` as well?
Not sure how common it is with webhooks, but I guess to potentially could:
https://community.zapier.com/code-webhooks-52/rss-feed-trigger-app-returns-not-acceptable-28640